css/src/style.scss

```diff
@@ -698,7 +698,7 @@
 				display: inline-block;
 				min-height: 24px;
 				margin: 13px 0;
-				width: calc(100% - 116px);
+				width: calc(100% - (41px + 16px + 45px + 45px + 14px)); /* checkbox (41px), status display (16px),  star (45px), pin (45px), safety distance (14px) */
```

Comment:
This looks bad. I guess we should use flexbox for the title of the details view. Can be done in a follow-up PR, though.